plugins/parser-morphlines/src/main/scala/com/stratio/sparkta/plugin/parser/morphline/MorphlineImplementation.scala

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
Since you has change the name on this PR. Could you rename it to something more explicit? Like KiteMorphlineImpl or something like that